webpack/components/Search/index.js

```diff
@@ -39,7 +39,7 @@ class Search extends Component {
           text: label.trim(),
         })),
       });
-    });
+    }).catch(() => {});
```

Comment:
@tstrachota This is empty to prevent the unhandled promise warning, but do we want to catch errors here?

Reply:
If something breaks it would produce an error on each type in the field, right? In this case I'd avoid propagating the errors as toast notifications but I think it would make sense to log them into console.